app/config/wallabag.yml

```diff
@@ -163,6 +163,20 @@ wallabag_core:
         - 'OpenDyslexicRegular'
         - 'Oswald'
 
+    client_side_rendered_sites:
+        proxy_url: https://single-file-proxy.your-domain.tld
+        proxy_all: true
```

Comment:
These values must be put in the `app/config/parameters.yml.dist` file instead. Not every one will have a single file proxy instance running.

Reply:
OK, fixed.